app/Model/ObjectTemplate.php

```diff
@@ -316,4 +316,41 @@ public function setActive($id)
         }
         return 1;
     }
+
+    public function getRawFromDisk($uuidOrName)
+    {
+        $templates = $this->readTemplatesFromDisk();
+        if (Validation::uuid($uuidOrName)) {
+            return $templates[$uuidOrName];
+        } else {
+            foreach ($templates as $uuid => $template) {
+                if ($template['name'] == $uuidOrName) {
+                    return $template;
+                }
+            }
+        }
+    }
+
+    private function readTemplatesFromDisk()
+    {
+        $objectsDir = APP . 'files/misp-objects/objects';
```

Comment:
Please move this to a constant to avoid storing this path here? See line 52 in function update where this *exact* same path string also appears, that could use the constant also

Reply:
There is a lot of duplication in the Model. This and the update process should be refactored.
I thought the feature was needed as fast as possible and that optimizations and refactoring for the whole model could wait.

Reply:
In my experience, duplication like this is rarely or never cleaned. It's far better to start clean instead of adding technical debt like this. This is only one string constant that can be used in two places within this file, should be easy, right?